code/game/objects/items/toys.dm

```diff
@@ -1650,3 +1650,18 @@ obj/item/toy/cards/deck/syndicate/black
 	icon_state = "conch"
 	use_action = "pulls the string"
 	possible_answers = list("Yes.", "No.", "Try asking again.", "Nothing.", "I don't think so.", "Neither.", "Maybe someday.")
+
+/*
+ *Fake cuffs (honk honk)
+ */
+
+/obj/item/restraints/handcuffs/toy
+	name = "handcuffs"
+	desc = "Toy handcuffs. Plastic and extremely cheaply made."
+	gender = PLURAL

```

Comment:
Since this is a subtype of handcuffs, all attributes from handcuffs will be inherited. Setting these vars is unneccessary if they're the same as regular handcuffs.

Reply:
Gotcha